<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeviceSelectDialog</class>
 <widget class="QDialog" name="DeviceSelectDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1922</width>
    <height>1314</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: black;
color: white;
}
QLabel{
font: 20pt &quot;Helvetica Neue&quot;;
}
QLabel#labelInfo{
font: 12pt &quot;Helvetica Neue&quot;;
}
QLineEdit{
font: 20pt &quot;Helvetica Neue&quot;;
color: #A9A9A9;
}
QFrame{
border: 0px;
}
QPushButton{
font: 12pt;
color: white;
}
QDialog{
border: 2px solid grey; 
border-radius: 200px;
}
QLineEdit{
border-top: 0px;
border-bottom: 2px solid grey;
}
QLabel#labelTitle{
border-bottom: 2px solid grey;
font: 30pt &quot;Helvetica Neue&quot;;
font-weight: 600
}
QFrame#frameDone{
background-color:#262626;
border-radius: 10px;
}
QFrame#frameBack{
background-color:#636363;
border-radius: 10px;
}
QListWidget{
font: 24pt &quot;Helvetica Neue&quot;;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frameDialog">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>57</y>
     <width>1800</width>
     <height>1200</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1800</width>
     <height>1200</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1922</width>
     <height>1314</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QFrame" name="frameTitle">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>180</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="labelTitle">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>402</width>
         <height>120</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>120</height>
        </size>
       </property>
       <property name="text">
        <string>DEVICE SELECTION</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameInfo">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frameAtherectomy">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>274</width>
         <height>313</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="labelAtherectomy">
          <property name="styleSheet">
           <string notr="true">text-decoration: underline;</string>
          </property>
          <property name="text">
           <string>ATHERECTOMY</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetAtherectomy">
          <property name="styleSheet">
           <string notr="true">QListView::item:selected{
background: rgb(99,99,99);
border-radius: 20px;
color: white;
}</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frameCto">
       <property name="geometry">
        <rect>
         <x>891</x>
         <y>0</y>
         <width>274</width>
         <height>313</height>
        </rect>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>40</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="labelCto">
          <property name="styleSheet">
           <string notr="true">text-decoration: underline;</string>
          </property>
          <property name="text">
           <string>CTO CROSSING</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListWidget" name="listWidgetCto">
          <property name="styleSheet">
           <string notr="true">border-left: 4px solid grey;</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="frameNext">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>100</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QFrame" name="frameBack">
       <property name="geometry">
        <rect>
         <x>1322</x>
         <y>20</y>
         <width>200</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonBack">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>&lt; BACK</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QFrame" name="frameDone">
       <property name="geometry">
        <rect>
         <x>1582</x>
         <y>20</y>
         <width>200</width>
         <height>27</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>60</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="leftMargin">
         <number>20</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>20</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="pushButtonDone">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>DONE</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonDone</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSelectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1249</x>
     <y>905</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonBack</sender>
   <signal>clicked()</signal>
   <receiver>DeviceSelectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>989</x>
     <y>905</y>
    </hint>
    <hint type="destinationlabel">
     <x>708</x>
     <y>514</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
